lib/connect.js

```diff
@@ -138,7 +138,8 @@ function connect(token, options) {
     preferredAudioCodecs: [],
     preferredVideoCodecs: [],
     realm: constants.DEFAULT_REALM,
-    signaling: SignalingV2
+    signaling: SignalingV2,
+    _useTwilioConnection: false
```

Comment:
What do you think about naming this property be `transport` and letting it be one of "sip" or "tcmp", defaulting to "sip"?

Reply:
@markandrus Generally I would agree, if this were a public ConnectOptions flag. Since its private and temporary, I don't know if its worth the effort to change it.